endpoint-edgecases/net_version.md

```diff
@@ -0,0 +1,9 @@
+# `net_version`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+Specification | Description 
+---|---
+1 |net_version MUST return the current network id of the client as type string|
+1.1 |net_version MUST consider a max network id of size FFFFFFFFFFFFFFFF or less|
```

Comment:
This would be 0xffff... Hex codes don't consider capitals

Reply:
max of X or less? what is less?

Reply:
I think this should be an information on type, not on chain ID spec